### DISCUSSION TITLE
Utiliser un export d'un autre fournisseur (RET)

### DISCUSSION THREAD

**Discussion by Fireyer** [JC144/EDF_Simulateur_Prix#86] [Q&A]
Bonjour,
D'abord un grand merci pour ce projet! Bravo. C'est super utile pour comparer factuellement! 

Mon problème, j'habite à Thônes en Haute savoie. Et nous avons un fournisseur unique local. La Régie Électrique de Thônes.
L'export de consommation que j’obtiens ne semble pas convenir. Pourriez vous me communiquer des csv en exemple pour que je fasse correspondre les champs. 
Je vous joint le CSV obtenu sur le site de la RET. Pourriez vous me donner des piste pour le rendre utilisable?
Merci d'avance
[export_consommations_energie.csv](https://github.com/JC144/EDF_Simulateur_Prix/files/14639420/export_consommations_energie.csv)

**Comment by JC144** [comment]
Bonjour,

C'est le seul fichier disponible ? Si vous avez un compteur Linky, vous pouvez peut-être récupérer un fichier plus précis sur le site d'Enedis ?

Le fichier actuel n'affiche que la consommation journalière. On ne pourra pas l'utiliser sur ce comparateur qui fait de l'analyse par tranche horaire (pour simuler précisément les heures creuses / pleines et la consommation sur les différents tarifs journaliers)
